pkg/network/admitter/binding_test.go

```diff
@@ -79,16 +94,18 @@ var _ = Describe("Validating network binding combinations", func() {
 
 var _ = Describe("Validating core binding", func() {
 	It("should reject a masquerade interface on a network different than pod", func() {
-		spec := &v1.VirtualMachineInstanceSpec{}
-		spec.Domain.Devices.Interfaces = []v1.Interface{{
-			Name:                   "default",
-			InterfaceBindingMethod: v1.InterfaceBindingMethod{Masquerade: &v1.InterfaceMasquerade{}},
-			Ports:                  []v1.Port{{Name: "test"}},
-		}}
-		spec.Networks = []v1.Network{{
-			Name:          "default",
-			NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "test"}},
-		}}
+		spec := &libvmi.New(
+			libvmi.WithName("testvmi"),
+			libvmi.WithInterface(v1.Interface{
+				Name:                   "default",
+				InterfaceBindingMethod: v1.InterfaceBindingMethod{Masquerade: &v1.InterfaceMasquerade{}},
+				Ports:                  []v1.Port{{Name: "test"}},
+			}),
+			libvmi.WithNetwork(&v1.Network{
+				Name:          "default",
+				NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "test"}},
+			}),
+		).Spec
 
 		validator := admitter.NewValidator(k8sfield.NewPath("fake"), spec, stubClusterConfigChecker{})
```

Comment:
Please consider the following alternative:
```suggestion
		vmi := libvmi.New(
			libvmi.WithName("testvmi"),
			libvmi.WithInterface(v1.Interface{
				Name:                   "default",
				InterfaceBindingMethod: v1.InterfaceBindingMethod{Masquerade: &v1.InterfaceMasquerade{}},
				Ports:                  []v1.Port{{Name: "test"}},
			}),
			libvmi.WithNetwork(&v1.Network{
				Name:          "default",
				NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "test"}},
			}),
		).Spec

		validator := admitter.NewValidator(k8sfield.NewPath("fake"), vmi.Spec, stubClusterConfigChecker{})
```

Same on the other occurrences bellow.

Reply:
Done !